code/src/components/TimeRange.js

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react'
```

Comment:
This is not used in the project, right? Lack of time or just changed you mind? 
You could keep it here if you thinking of implementing it, but maybe add a comment in this file that it's not used or something like that?